spyder_line_profiler/spyder/plugin.py

```diff
@@ -0,0 +1,128 @@
+# -*- coding: utf-8 -*-
+# ----------------------------------------------------------------------------
+# Copyright © 2021, Spyder Line Profiler contributors
+#
+# Licensed under the terms of the MIT license
+# ----------------------------------------------------------------------------
+
+"""
+Spyder Line Profiler 5 Plugin.
+"""
+
+# Third-party imports
+from qtpy.QtCore import Signal
```

Comment:
To add `Qt` import needed for the new context argument for the run action:
```suggestion
from qtpy.QtCore import Qt, Signal
```